tapfreighter/wallet.go

```diff
@@ -910,6 +910,16 @@ func createAndSetInput(vPkt *tappsbt.VPacket, idx int,
 	}
 	vPkt.SetInputAsset(idx, assetInput.Asset)
 
+	inputAltLeaves, err := assetInput.Commitment.FetchAltLeaves()
+	if err != nil {
+		return fmt.Errorf("cannot fetch alt leaves from input: %w", err)
+	}
+
+	err = vPkt.Inputs[idx].SetAltLeaves(inputAltLeaves)
```

Comment:
Hmm, I thought we wouldn't start producing alt leaves just yet, only in the next version? Or I guess since the alt leaves would only be in the root output's commitment, a non-interactive receiver wouldn't come into contact with them yet.

Reply:
So IIUC, we aren't quite _producing_ them yet, but we always make sure that if they're present, they're passed along/validated.

Reply:
We'll also need to do this (`SetAltLeaves`) once we're constructing a `vOut` when prepping to send.